src/codeu/chat/client/simplegui/NewAccount.java

```diff
@@ -0,0 +1,98 @@
+package codeu.chat.client.simplegui;
+
+import java.awt.Button;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.HashMap;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JPasswordField;
+import javax.swing.JTextField;
+
+import codeu.chat.UserMap;
+
+@SuppressWarnings("serial")
+/**
+ * @author  Troy Jones
+ * @date    5/21/17
+ * @brief   Contains all GUI necessary to execute the frame to 
+ *          create a new account.
+ */
+public class NewAccount extends JFrame {
+
+    UserMap userMap;
```

Comment:
members of a class should be private unless there's a very good reason otherwise